### PR TITLE
Introduce Changelog files for connect, connect-extension-protocol and…

### PR DESCRIPTION
**Pull request by wirednkod**
… extension

### CODE REVIEW

**Comment by tomaka** [comment]
This needs to be merged right after a new version is published, or contain the list of unreleased changes

**Comment by wirednkod** [comment]
> This needs to be merged right after a new version is published, or contain the list of unreleased changes

When next PR is merged (e.g. logs) then we need to add this to the changelog (under unreleased). Thats why I think its a good idea to merge it prior to any other merge - so that files are there for updating

**Comment by tomaka** [comment]
I also don't think it's worth saying that we've added a changelog. It's pretty clear when you see that the changelog starts that there was no changelog before

**Comment by tomaka** [comment]
> When next PR is merged (e.g. logs) then we need to add this to the changelog (under unreleased). Thats why I think its a good idea to merge it prior to any other merge - so that files are there for updating

What I mean is that you need to add a line about https://github.com/paritytech/substrate-connect/pull/842/

File: packages/connect/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 
-### Added
+# Fixed
 
-- Changelog
+- Add missing subscription/unsubscription methods ([#850](https://github.com/paritytech/substrate-connect/pull/850/))
+- Wrong unsubscription JSON-RPC methods being used when calling `ProviderInterface.disconnect()` ([#842](https://github.com/paritytech/substrate-connect/pull/842)).
```
Comment (tomaka) [inline packages/connect/CHANGELOG.md:8]: There's no need for it to be two lines
Users don't care that we fixed a bug introduced in between two versions

```suggestion
- Fixed wrong unsubscription JSON-RPC methods being used when calling `ProviderInterface.disconnect()` ([#842](https://github.com/paritytech/substrate-connect/pull/842)).
```

File: projects/extension/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,6 @@
 
 ## [Unreleased]
 
-### Added
+# Removed
 
-- Changelog
+- Usage of the `SmoldotHealth` interface ([#845](https://github.com/paritytech/substrate-connect/pull/845))
```
Comment (tomaka) [inline projects/extension/CHANGELOG.md:7]: This is an internal change, users don't care about that

```suggestion
```

File: projects/extension/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Changelog
+
+## [Unreleased]
+
+# Removed
+
```
Comment (tomaka) [inline projects/extension/CHANGELOG.md:6]:
```suggestion
```

**Review by tomaka — APPROVED** [review]

**Pull request merged**